bumble/device.py

```diff
@@ -906,6 +906,30 @@ async def get_phy(self):
     async def request_remote_name(self):
         return await self.device.request_remote_name(self)
 
+    async def read_remote_feature(self) -> None:
```

Comment:
This method could return the feature mask, just like `get_phy` returns the PHY values ,or `get_rssi` returns the RSSI value.

Reply:
Sure, it makes sense to separate Classic and LE into two methods.